webplugin/js/app/conversation/mail-parser/dom-service.js

```
@@ -0,0 +1,84 @@
+            });
+        }
+    }
+
+    static getLoadingSpinner() {
+        return `<div class="km-eml-loading">Loading...</div>`;
+    }
+
+    static getFromMarkup(from = {}) {
+        return `<div class="km-email-from-heading" title="${from.address}"><strong class="km-email-from">${from.name}</strong> <p>&lt;${from.address}&gt;</p></div>`;
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)